hbp_knowledge/receptors/zoli2018.bel

```diff
@@ -83,214 +92,337 @@ a(MESH:Acetylcholine) -- bp(MESH:Apoptosis)
+r(HGNC:CHRFAM7A,loc(MESH:Lymphocytes))
+#initially the location was set as "Leukemia, Large Granular Lymphocytic" but it did not match the evidence. Hence, anatomy was set and location was changed.
+UNSET Anatomy
+UNSET Confidence
 
 SET Evidence ="Functional studies have shown that α7dup, expressed in oo- cytes, acts as a dominant negative regulator of α7 nAChR activity by means of a mechanism involving a reduction in the number of functional α7 nAChRs incorporated into the oocyte surface "
+SET Confidence = "High"
 p(HGNC:CHRFAM7A,loc(MESH:Oocytes)) -| act(p(HGNC:CHRNA7,loc(MESH:Oocytes)))
+UNSET Confidence
 
 SET Evidence =" α4β2* subtypes, in which the presence of different accessory subunits changes their pharmacological and biophysical properties, and their sensitivity to allosteric modulators and up-regulation by nicotine "
-a(MESH:Nicotine) -| act(complex(p(HGNC:CHRNA4),p(HGNC:CHRNB2)))
+SET Confidence = "Medium"
+a(MESH:Nicotine) -> act(complex(p(HGNC:CHRNA4),p(HGNC:CHRNB2)))
+#the evidence says upregulation of alpha4beta2 by Nicotine
+UNSET Confidence
 
 SET Evidence ="nAChRs are expressed also at the somatodendritic postsynaptic site, where they regulate neuron depolarisation, firing and long- term potentiation [9]. Moreover these receptors are also in- volved in proliferation, differentiation and migration of neu- ral progenitors"
+SET Confidence = "Medium"
+p(MESH:"Receptors, Nicotinic") -- bp(GO:"depolarization of postsynaptic membrane")
+#added on the basis of evidence
 p(MESH:"Receptors, Nicotinic") -- bp(GO:"action potential initiation")
 p(MESH:"Receptors, Nicotinic") -- bp(HBP:"cell population proliferation")
 p(MESH:"Receptors, Nicotinic") -- bp(GO:"cell differentiation")
-p(MESH:"Receptors, Nicotinic") -- bp(GO:"cell differentiation")
+UNSET Confidence
 
 SET Evidence ="nAChRs contribute to cognitive function, and changes in their number and/or func- tion are associated with various pathological conditions such as cognitive disorders, anxiety, depression, Alzheimer’s and Parkinson’s disease, pain and epilepsy "
+SET Confidence = "High"
 p(MESH:"Receptors, Nicotinic") -- path(MESH:"Cognition Disorders")
 p(MESH:"Receptors, Nicotinic") -- path(MESH:Anxiety)
 p(MESH:"Receptors, Nicotinic") -- path(MESH:Depression)
 p(MESH:"Receptors, Nicotinic") -- path(MESH:"Alzheimer Disease")
 p(MESH:"Receptors, Nicotinic") -- path(MESH:"Parkinson Disease")
 p(MESH:"Receptors, Nicotinic") -- path(MESH:Pain)
 p(MESH:"Receptors, Nicotinic") -- path(MESH:Epilepsy)
+UNSET Confidence
 
 SET Evidence ="nAChRs are particularly important in two critical periods of brain life: early pre- and post-natal circuit formation, and age-related cell degeneration. They are involved in neuronal survival, as it has been shown that nicotinic agonists are neu- roprotective in in vivo and in vitro models"
+SET Confidence = "Medium"
 p(MESH:"Receptors, Nicotinic") -- a(MESH:Neurons)
-a(MESH:Nicotine) -| a(MESH:Neurons)
+a(MESH:Nicotine) -> a(MESH:Neurons)
```

Comment:
you are right, but I would use the downregulated process "neuron death" instead, as this is the exact process that is influenced here
```suggestion
a(MESH:Nicotine) -| bp(GO:" neuron death")
``